### DISCUSSION TITLE
Dealing with cannibalization between shops

### DISCUSSION THREAD

**Discussion by Johannes96** [google/lightweight_mmm#95] [Q&A] [answered]
Hi team,

I have a question regarding the cannibalization between individual shops. Hierarchical regression can look at several shops at once, but does it also consider cannibalization effects between these shops? For example one shop invests heavily in ads, this might increase the sales of this particular shop but maybe not the overall-sales. It might just shift the sales from one shop to another. Is your hierarchical model able to handle this? 

Regrads,
Johannes

**Comment by michevan** [comment, marked as answer]
Hi Johannes

If I'm understanding this correctly, you can measure media spend and sales independently for each region here but the regions are small enough that there's significant risk of users seeing ads in one region but then making a purchase in a different region? I think I would consider this something like a "spillover effect", and usually that would be a sign that you need to use larger regions so that the media impressions and their corresponding sales are mostly contained within the same regions.

To your point though, I guess if you have spillover effects like this, a hierarchical model might be able to help in some cases. Some of the heterogeneity between regions introduced by the spillover might be able to be captured by the hierarchical nature of the beta parameters. I wouldn't recommend pursuing this too deeply, though; if spillover is a major issue and you don't want to change the size of your regions, you'd probably want to incorporate this cannibalization explicitly into your model somehow in order to measure it more precisely.
